features/all-positions/AllPositions.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Paper, Container, Typography } from "@material-ui/core";
+
+import Token from "../../containers/Token";
+import Connection from "../../containers/Connection";
+
+const AllPositions = () => {
+    const { symbol: tokenSymbol } = Token.useContainer();
+
+    if (tokenSymbol === null) {
+        return (
+          <Container>
+            <Box py={2}>
+              <Typography>
+                <i>Please first select an EMP from the dropdown above.</i>
+              </Typography>
+            </Box>
+          </Container>
+        );
+    }
+
+    const getEstimatedPrice = () => {
+        if (tokenSymbol?.includes("yCOMP")) {
+            return 187;
+        } else {
+            return 0.0259;
+        }
+    }
```

Comment:
```suggestion
    const estimatedPrice = tokenSymbol?.includes("yCOMP") ? 187 : 0.0259
```

Of course this also means you can't call it as a function below.

Reply:
This is also a temporary hard coded return val. I will soon make this a dynamic call to an off-chain price feed